src/DynamoDbQueryBuilder.php

```diff
@@ -551,10 +569,15 @@ public function save()
         return array_get($result, '@metadata.statusCode') === 200;
     }
 
+    public function get($columns = [])
+    {
+        return $this->all($columns);
+    }
+
     public function all($columns = [])
     {
         $limit = isset($this->limit) ? $this->limit : static::MAX_LIMIT;
-        return $this->getAll($columns, $limit);
+        return $this->getAll($columns, $limit, !isset($this->limit));
```

Comment:
hm why do we need this change?

Reply:
This prevents the iterator from being called, which does not set lastEvaluatedKey ... when using the iterator, `afterKey` failed to work... this just bypasses that when we explicitly set a limit (which would result in us expecting a lastEvaluatedKey being returned)